src/daft-parquet/src/python.rs

```diff
@@ -68,20 +68,20 @@ pub mod pylib {
         schema: arrow2::datatypes::SchemaRef,
         all_arrays: Vec<ArrowChunk>,
         num_rows: usize,
-        pyarrow: &PyModule,
+        pyarrow: Bound<'_, PyModule>,
     ) -> PyResult<PyArrowParquetType> {
         let converted_arrays = all_arrays
             .into_iter()
             .map(|v| {
                 v.into_iter()
-                    .map(|a| to_py_array(a, py, pyarrow))
+                    .map(|a| to_py_array(a, py, pyarrow.clone()))
                     .collect::<PyResult<Vec<_>>>()
             })
             .collect::<PyResult<Vec<_>>>()?;
         let fields = schema
             .fields
             .iter()
-            .map(|f| field_to_py(f, py, pyarrow))
+            .map(|f| field_to_py(f, pyarrow.clone()))
```

Comment:
should this just take a bound reference to the module?

Reply:
Yep you're right. Changed